license_server/license_contract.proto

```diff
@@ -62,19 +64,61 @@ message CheckLicenseUsageRequest {
     uint64 channel_id = 2;
     string service_id = 3;
     uint64 price_in_cogs = 4;
-    bool  isDynamicPrice = 5;
+    bool isDynamicPrice = 5;
 
 }
 
 message LicenseCreateRequest {
     CallerAuthentication auth = 1;
-    //the channel exclusively created for this license_server needs to be passed
+    //The channel exclusively created for this license_server needs to be passed
+    //Channel Id , will automatically tell you the org id and the group id
     uint64 channel_id = 2;
+
     //you can pass the list of Authorized addresses to also be allowed to use the License.
-    repeated string user_addresses = 4;
```

Comment:
The number has changed and also 5 is missing. Is that expected?

Reply:
this was a typo, fixed now